### ISSUE TITLE
Possible to rename local file

### ISSUE THREAD

**Issue by Phx01** [neffo/bing-wallpaper-gnome-extension#31]
Normally I used my Windows VM with Bing Desktop to download the images.  However, this neat extension for GNOME is neat and saves me some time and resources.  One thing, I would like to see, however, is to be able to change the naming convention of the local files.  Right now it is <date>-<EXIF_Image_Description>-<resolution>.

In bug #29  one needed to change a line inside "extensions.js" to make v15 work.  If it would be equally as trivial to change it to only include the date omitting the rest, it would be enough for me to know where to do it as I am not too familiar with JavaScript and therefore did not find where the local filename is defined.

**Comment by Phx01** [comment]
Finally got to test it and it worked.  As it turned out I found that section before, but nothing changed when I altered it then.  For some reasons  I needed to restart the entire computer as simply restarting the extension and/or logging out and back in did not work.

Thanks a lot!  :-)

**Comment by neffo** [comment]
Yeah, the GNOME shell itself needs to be restarted, there are a couple of ways to do that depending on your OS and whether it's running Xorg or Wayland.

If your OS is running Xorg (Ubuntu, for example) restarting the shell can be performed by pressing Alt-F2 and typing 'r' in the command box that appears. If your OS is running Wayland (Fedora, for example) restarting the shell requires you to log-out/log-in again. Restarting the computer works in either case.
